features/channel/managing_channels/editing_shop_billing_data_on_channel.feature

```diff
@@ -10,13 +10,13 @@ Feature: Editing shop billing data on channel
         And channel "Web Store" billing data is "Ragnarok", "Pacific Coast Hwy", "90806" "Los Angeles", "United States" with "1100110011" tax ID
         And I am logged in as an administrator
 
-    @ui
+    @ui @api
     Scenario: Editing shop billing data on channel
         When I want to modify a channel "Web Store"
         And I specify company as "Götterdämmerung"
         And I specify tax ID as "666777"
         And I specify shop billing address as "Valhalla", "123" "Asgard", "United States"
-        And I save my changes
+        And I save my change of shop billing data
```

Comment:
```suggestion
        And I save my change of shop billing data
```
In my opinion, this change is unnecessary. What was an intention here?

Reply:
Guessing a naming conflict in the API context.
Extracting it into a separate scenario only for API would probably be the simplest workaround since smth like `saving shop billing data changes` doesn't really exist on UI.